packages/frontend/src/app/repository/repository.component.html

```diff
@@ -50,10 +50,8 @@
     </div>
 </div>
 
-
-<!-- REPOSITORIES LIST -->
-<div class="build-list-container">
-    <app-build-list [builds]="mockBuilds" [repoName]="repoName" [orgName]="orgName"></app-build-list>
```

Comment:
Let's discuss details of which builds it will return, and whether we will leave scrolling / pagination until after we have MVP. BTW the api endpoint `/repo/:orgname/:reponame` should include all the information used for this page. Looking at response.builds[0].tests would return the limited test information for the part below

Reply:
Yes. Having the tests in _builds[0].tests_ makes sense. LGTM.
Yes, I think that we can leave the pagination for after the MVP. @bcoe ? No pagination here meaning that all tests will be fetched once (the pagination in the frontend will still be there).